examples/ziti-end-to-end/README.md

```diff
@@ -0,0 +1,172 @@
+# Setting up an End-to-End Ziti network
+## Prerequisites:
+
+Go 1.19 installed and setup: [download](https://go.dev/doc/install). You're going to want to set you `$GOPATH` env variable which can be anywhere on your system. This is where modules, packages, and installed binaries will end up. Putting `$GOPATH/bin` in your path will help with running the executables later.
+
+Ziti Binaries and relevant configs.
+
+[OpenZiti](https://github.com/openziti/ziti/tree/main) contains downloads for the latest builds or you can build it yourself. There are many example configs [in the etc dir](https://github.com/openziti/ziti/tree/main/etc).
+
+I have set up helpful environment variable for how I chose to get setup. I cloned the Ziti repository for local builds and set `$ZITI_SOURCE` to that directory. I then went into that directory and ran 
+```
+go install ./...
+```
+
+This will output the binary to `$GOPATH/bin`. I then do the same thing with this repository to install the go-httpbin binaries
+
+## Starting up the Ziti Network:
+
+### Controller Setup
+
+First we run the ziti controller with the arguent being which ever config file we chose. For this example I am using the `ctrl.with.edge.yml` file bundled for examples in the ziti repository.
+```
+ziti-controller run --log-formatter pfxlog $ZITI_SOURCE/ziti/etc/ctrl.with.edge.yml
+```
+
+We then need to initialize the controller with an admin user. For ease of testing I just used `admin admin admin` for my short local test.
+
+```
+ziti agent controller init <username> <password> <name-of-user>
+```
+
+You can then login to the edge via `ziti edge login` and putting in your credentials.
```

Comment:
this might work in a pristine dev env but I found it necessary to specify the edge-mgmt server URL.

```bash
❯ ziti edge login localhost:1280
Untrusted certificate authority retrieved from server
Verified that server supplied certificates are trusted by server
Server supplied 7 certificates
Trust server provided certificate authority [Y/N]: y
Server certificate chain written to /home/kbingham/.config/ziti/certs/localhost
Enter username: admin
Enter password: 
Token: 116a4b7b-120f-4b70-9c47-b09a1f422731
Saving identity 'staging-nfadmin-nfziti-nfctl-demo-ringed-chestnut' to /home/kbingham/.config/ziti/ziti-cli.json
```

The last line looks like a bug with session mgmt, but my session behaves as expected despite the unexpected value of the `default` property in the `ziti-cli.json` file.

Reply:
I did try and do this from a fresh environment, so would have missed any session edge cases like this. I'll try and dig into it more